crates/languages/src/typescript/runnables.scm

```diff
@@ -1,4 +1,4 @@
-; Add support for (node:test, bun:test and Jest) runnable
+; Add support for (node:test, bun:test, Jest and Deno.test) runnable
```

Comment:
I wonder if the tasks are gonna have a `vitest: `/`jest: ` tasks in the UI; if so, that might be a bit confusing but we can prolly roll with it anyways.

Reply:
No idea and I'm not sure what you mean exactly, I tested with deno and I didn't see anything confusing

Would changing js-test tag to something different like deno-test help?

Reply:
Sorry, I meant "`vitest/jest` prefix in the UI". I'm gonna take your word on this one.